File: strings/kleberbarros/1551.py
N = int(input())
strgv = [input().replace(',', '').replace(' ', '') for i in range(N)]

alfav = [0]*N

for i in range(N):
    alfav[i] = len(set(strgv[i]))

for i in range(N):
    if(alfav[i]==26):
        print('frase completa')
    elif(alfav[i] >= 13):
        print('frase quase completa')
    else:
        print('frase mal elaborada')

File: strings/kildere/1551.java
import java.util.Scanner;

public class Main {
	
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		int n = Integer.parseInt(scan.nextLine());
		String s;
		int tem;
		
		for (int i = 0; i < n; i++) {
			tem = 0;
			s = scan.nextLine();
			
			for (char j = 'a'; j <= 'z'; j++) {
				if(s.split((""+j)).length > 1){//tiver
					tem++;
				}
			}
			
			//System.out.println(tem);
			
			if(tem == 26){
				System.out.println("frase completa");
			}else if(tem >= 13){
				System.out.println("frase quase completa");
			}else{
				System.out.println("frase mal elaborada");
			}
		}
	}
}